### PR TITLE
fix user role dashboard test

### PR DESCRIPTION
**Pull request by rohan2794**
Fix for user role dashboard test case in dashboard section

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -27,12 +27,12 @@ def driver(request):
...
-            desired_capabilities={
-                "browserName": request.config.getoption("--driver")
-            })
-
+    # browser = webdriver.Remote(
+    #         command_executor="http://" + hub + ":4444/wd/hub",
+    #         desired_capabilities={
+    #             "browserName": request.config.getoption("--driver")
+    #         })
+    browser = webdriver.Chrome(executable_path='/home/rohan/Downloads/chromedriver')
```
Comment (nrusinko) [inline conftest.py:35]: @rohan2794 please revert this change.

Comment (rohan2794) [reply conftest.py]: change reverted with some other logout fix

File: main/pages/BasePage.py [hunk 2]
```diff
@@ -101,3 +104,18 @@ def verify_email_present(self, email):
...
         assert "test" in data
+
+    def move_to_element(self, locator):
+        element = self.wait_element_visible(locator)
+        actions = ActionChains(self.driver)
+        actions.move_to_element(element)
+        # perform the operation on the element
+        actions.perform()
+
+    def logout(self):
```
Comment (nrusinko) [inline main/pages/BasePage.py:115]: @rohan2794 do we really need to have this in a BasePage?

Comment (rohan2794) [reply main/pages/BasePage.py:115]: @nrusinko  at some steps where SidePanel Page reference is  not present , there i am calling logout() on BasePage reference
